Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1196,6 +1191,9 @@
     <PackageReference Include="Microsoft.VisualStudio.Threading">
       <Version>15.5.13-beta</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.VsInteractiveWindow">
+      <Version>2.3.0</Version>
+    </PackageReference>
```

Comment:
This is roughly equivalent to a packages.json entry?

Reply:
Yes, the 'new' way to add nuget packages...